<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>智能家庭控件</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>240</y>
      <width>147</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>窗帘控制</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_motor_back">
         <property name="text">
          <string>上升</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_motor_stop">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_motor_front">
         <property name="text">
          <string>下降</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>240</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>温度</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="tempNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>湿度</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="humiNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>光强度</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lightNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_light">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>200</y>
      <width>93</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动调光</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>200</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>灯光亮度调节</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>烟雾报警</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>盗窃报警</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSmoke">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSteal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>91</y>
      <width>39</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>总开关</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_up">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>91</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_down">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>91</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_log">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>360</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>500</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>日志窗口      更多日志请查看‘log.txt’</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
